showcase/app/routes/page-components/advanced-table.ts

```diff
@@ -24,149 +27,30 @@ export const clone = <T>(arr: T[]): T[] => {
-    | string;
-  service?:
-    | {
-        text: string;
-        colspan?: number;
-      }
-    | string;
-  description?: string;
-  email?: string;
-}
-
-interface Policy {
-  id: number;
-  name: string;
-  description: string;
-  status: string;
-  isOpen?: boolean;
-  children?: Policy[];
-}
-
-interface PolicyCustom {
-  id: number;
-  name: string;
-  description: string;
-  status: string;
-  isOpen?: boolean;
-  data?: Policy[];
-}
-
 export default class PageComponentsAdvancedTableRoute extends Route {
-  async model() {
-    const responseMusic = await fetch('/api/folk.json');
-    const responseUserData = await fetch('/api/mock-users.json');
-    const responseClusters = await fetch('/api/mock-clusters-with-status.json');
-    const responseSelectableData = await fetch(
-      '/api/mock-selectable-data.json',
-    );
-    const responseManyColumns = await fetch('/api/mock-many-columns.json');
-    const responseSpanningManual = await fetch(
-      '/api/mock-spanning-cells-manual.json',
-    );
-    const responseNested = await fetch('/api/mock-nested-rows.json');
-    const responseNestedCustom = await fetch(
-      '/api/mock-nested-rows-custom.json',
-    );
-
-    const { data: music } = (await responseMusic.json()) as Record<
-      'data',
-      MusicEntity[]
-    >;
-    const userData = (await responseUserData.json()) as User[];
-    const clusters = (await responseClusters.json()) as Cluster[];
-    const selectableData =
-      (await responseSelectableData.json()) as SelectableItem[];
-    const manyColumns =
-      (await responseManyColumns.json()) as UserWithMoreColumns[];
-    const spanningManualData =
-      (await responseSpanningManual.json()) as SpanningEntity[];
-    const nestedData = (await responseNested.json()) as Policy[];
-    const nestedDataCustom =
-      (await responseNestedCustom.json()) as PolicyCustom[];
-
+  model() {
     return {
-      music: music.map((record) => ({ id: record.id, ...record.attributes })),
-      userDataShort: clone(userData.slice(0, 5)),
+      music: folkMusic,
+      userDataShort: clone(users.slice(0, 5)),
       clusters,
-      spanningManualData,
-      selectableData,
-      selectableDataDemo1: clone(selectableData),
-      selectableDataDemo2: clone(selectableData),
-      userData,
-      userDataDemo3: clone(userData.slice(0, 16)),
+      spanningManualData: spanningCells,
+      selectableData: selectableItems,
+      selectableDataDemo1: clone(selectableItems),
+      selectableDataDemo2: clone(selectableItems),
+      userData: users,
+      userDataDemo3: clone(users.slice(0, 16)),
       userDataDemo4: clone(
-        userData.slice(0, 4).map((user) => ({ ...user, isAnimated: false })),
+        users.slice(0, 4).map((user) => ({ ...user, isAnimated: false })),
       ),
-      manyColumns,
-      nestedData,
-      nestedDataCustom,
+      manyColumns: userWithMoreColumns,
+      nestedData: policies,
+      nestedDataCustom: policies.map((policy) => {
```

Comment:
[todo] One of the item, in the previous JSON, had `isOpen: true` so the VRT test would test also an opened nested row, you should add it back in this logic here (see Percy diff)
<img width="1175" height="316" alt="screenshot_5268" src="https://github.com/user-attachments/assets/a7849997-d2ef-40a4-95b6-a5e22c8e88b5" />